package.json

```diff
@@ -100,5 +100,8 @@
     "stylelint-scss": "1.5.1",
     "webpack": "^3.3.0",
     "webpack-dev-server": "^2.5.1"
+  },
+  "dependencies": {
+    "terra-grid": "^3.5.1"
```

Comment:
was this added on purpose? I don't think our root package.json should have dependencies should it?